go/genesis/genesis_test.go

```diff
@@ -412,6 +436,53 @@ func TestGenesisSanityCheck(t *testing.T) {
 	d.Registry.Entities = []*entity.SignedEntity{signedTestEntity}
 	require.NoError(d.SanityCheck(), "test entity should pass")
 
+	// NOTE: We will break the staking part of testDoc and gradually fix it to
+	// test all related registry & staking genesis checks.
+	d = *testDoc
+	d.Staking.Ledger = make(map[signature.PublicKey]*staking.Account)
+	d.Staking.Delegations = make(map[signature.PublicKey]map[signature.PublicKey]*staking.Delegation)
+
+	d.Registry.Entities = []*entity.SignedEntity{signedTestEntity}
+	err = d.SanityCheck()
+	require.NotNil(err, "entity without an account should be rejected")
+	require.Regexp("no account associated with entity with ID", err)
+
+	// Add account for test entity to staking ledger.
+	d.Staking.Ledger[validPK] = &staking.Account{
+		General: staking.GeneralAccount{
+			Balance: *quantity.NewQuantity(),
```

Comment:
sometimes `*quantity.NewQuantity()`, sometimes `stakingTests.QtyFromInt(0)`

Reply:
I've updated the code to consistently use `*quantity.NewQuantity()`.